aten/src/ATen/test/cuda_distributions_test.cu

```diff
@@ -22,7 +22,43 @@ __global__ void expected_randoms(float* x, uint64_t counter_offset) {
   }
 }
 
-TEST(DistributionsTest, TestPhiloxIncrementSmallTensor) {
```

Comment:
Was there a substantive change to this file, or did you just refactor some duplicated code into a function?

Reply:
Just refactored duplicated code. Nothing else changed.